app/helpers/admin_helper.rb

```diff
@@ -15,21 +15,8 @@ def active_booking
 		booking["group"]
 	end
 
-	def check_lock_state
-		#Gets all lockstates orders by creation and gets the latest one
-		latest_lock = LockState.order(:created_at).last
-		#if exp date is later than current time
-		if latest_lock.present? and latest_lock.expiration_date >= Time.now
-			#checks lock state
-			if latest_lock.state.eql?("locked")
-				@is_locked = true
-				#call to active booking to get @group (can this be done better?)
-				active_booking
-				unless current_user.in_group?(@group.to_s) || current_user.admin?
-					render :file => "app/views/lights/lock.html", :status => :unauthorized
-				end
-			end
-		end
-		@is_locked
-	end
+	def is_locked
```

Comment:
Minor comment, conventions, methods that return boolean value in ruby should have a question mark at the end like so: `is_locked?`